language-plutus-core/src/Language/PlutusCore/CBOR.hs

```diff
@@ -44,13 +45,27 @@ However, having this flexibility allows us to encode e.g. PLC with substantial a
 for testing.
 -}
 
+
+{- Encode/decode constructor tags using encodeWord and decodeWord.  We
+   previously used encodeTag/decodeTag here, but that was wrong: those
+   are for use with a fixed set of CBOR tags with predefined meanings
+   which we shouldn't interfere with.  Note that encodeWord will only
```

Comment:
I think it's a bit confusing to phrase this note in terms of what we used to do. I do think it's helpful to have a warning about correct usage of tags for constructors - make it a proper note and maybe reference the `serialise` library.